point_viewer_grpc_proto_rust/src/proto.proto

```diff
@@ -51,3 +53,17 @@ message GetPointsInBoxRequest {
 message GetPointsInBoxReply {
   repeated point_viewer.proto.Vector3f points = 1;
 }
+
+message GetPointsInFrustumRequest {
```

Comment:
Is this the most natural representation of a frustum? Personally, I'd find a Matrix4d for the view frame (view_position + view_direction + view_up) more natural, and separate horizontal_fov and vertical_fov expressed as tan(φ) preferable:
```
\    |____/
 \   | φ /
  \  |  /
   \ | /
    \|/
```
Hope the drawing helps.

Reply:
The matrix is not constraint though: it might represent another kind of frustum then a view frustum. The API as presented is harder to misuse I think.

We will add a function that takes a generic Matrix eventually.

Reply:
Ah, I'm not suggesting to pass a projection matrix, only a coordinate transform that determines which way the frustum is looking (including which way is up), and from where – you're right that passing any Matrix4 is overly general. How about translation + rotation quaternion? That's very close to what people are already working with.

Thoughts on the suggestion for the FOV units/representation?

Reply:
I have no preference here. I defer to @thomasschiwietz who knows the most about geometrical representation.

Reply:
translation + quaternion sounds like a good idea for people working with poses. if there are no customers for position + dir + up then let's remove it. fov in radians.